manim/utils/caching.py

```diff
@@ -37,7 +38,11 @@ def wrapper(self, scene, *args, **kwargs):
             )
             if self.file_writer.is_already_cached(hash_play):
                 logger.info(
-                    f"Animation {self.num_plays} : Using cached data (hash : %(hash_play)s)",
+                    f"Animation {self.num_plays} : "
+                    "Using cached data (hash : %(hash_play)s)",
+                    # an f-string is being used here; couldn't
+                    # the format dictionary be replaced with a
+                    # second interpolation?
```

Comment:
That's a fair question, and the answer is no. 

This is for logs tests of the librairy, as hashes are inconsitent over the runs we want to avoid having them in the logs. 
This is part of the trick to remove them :)